salt/metalk8s/kubeadm/init/kubelet-start/configured.sls

```diff
@@ -0,0 +1,51 @@
+{%- from "metalk8s/map.jinja" import kubelet with context %}
+
+Create kubelet service environment file:
```

Comment:
Missing makedirs or creating the parent directories

Reply:
This directory is created by the kubelet package I think but I can add a makedirs just "au cas où"